.env.example

```
@@ -134,4 +134,30 @@ UPLOAD_MAX_FILE_SIZE_MB=10
+# To force a specific provider, set RECEIPT_SCAN_PROVIDER to 'gemini' or 'ollama'.
+RECEIPT_SCAN_PROVIDER=
+
+# Gemini API key (used as fallback or when RECEIPT_SCAN_PROVIDER=gemini)
+GEMINI_API_KEY=
+GEMINI_MODEL=gemini-3.5-flash-lite
+
+# Ollama options. Set OLLAMA_MODEL to bypass auto-detection and force a specific tag.
+# Auto-detection probes the Ollama-native /api/tags endpoint, so other
+# OpenAI-compatible servers (LM Studio, vLLM, llama.cpp) won't be detected —
+# point OLLAMA_BASE_URL at them and set OLLAMA_MODEL explicitly.
+OLLAMA_BASE_URL= # http://localhost:11434
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)